src/ClassicUO.Client/LegionScripting/API.py

```
@@ -24,293 +24,1376 @@
+def ToggleScript(scriptName: str):
+    """
+     Toggle another script on or off.
+     Example:
+     ```py
+     API.ToggleScript("MyScript.py")
+     ```
+    
+    """
+    pass
+
+def AddMapMarker(name: str, x: int = int.MaxValue, y: int = int.MaxValue, map: int = int.MaxValue, color: str = "purple"):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)